src/sklearn_knn/original/_gnn.py

```diff
@@ -1,15 +1,12 @@
-from sklearn.pipeline import Pipeline
-
-from .._base import IDNeighborsClassifier, KnnPipeline
+from .._base import IDNeighborsClassifier
 from ._cca_transformer import CCATransformer
 
 
-class GNN(KnnPipeline):
-    def _get_pipeline(self):
-        transformer = CCATransformer()
-        classifier = IDNeighborsClassifier(
-            n_neighbors=self.n_neighbors,
-            weights=self.weights,
-        )
-        steps = [("transform", transformer), ("classifier", classifier)]
-        return Pipeline(steps)
+class GNN(IDNeighborsClassifier):
+    def fit(self, X, y, spp=None):
+        X = CCATransformer().fit_transform(X, spp=spp)
+        return super().fit(X, y)
+    
+    def predict(self, X, spp=None):
+        X = CCATransformer().fit_transform(X, spp=spp)
+        return super().predict(X)
```

Comment:
Nope, I agree

Reply:
I'll go ahead and do this and push to this branch

Reply:
I went ahead and moved `_gnn` out of `original` and into the root because I was getting confused by having 2 copies. I left the CCA stuff for now, but I'm fine with whatever you'd like to do with those, whether it's moving them into the root, making a `transformers` module, or just leaving them as is and worrying about them in another PR.